modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/resources/com/liferay/dynamic/data/mapping/dependencies/upgrade.v5_3_2/fragment-entry-browser-sniffer-content.html

```diff
@@ -0,0 +1,19 @@
+<div class="fragment-1">
+[#assign normalizer = serviceLocator.findService("com.liferay.portal.kernel.util.FriendlyURLNormalizerUtil") /]
+
+[#assign browserSnifferCaseOne = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil") /]
+
+[#assign browserSnifferCaseTwo = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil") ]
+
+[#assign
+browserSnifferCaseThree = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil")
+variableACaseThree = "variable A test"
+variableBCaseThree = "variable B test"
+/]
+
+[#assign
+browserSnifferCaseFour = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil")
+variableACaseFour = "variable A test"
```

Comment:
@alberttgomes, do we really need to use `variableACaseFour` instead of only `variableA`?